README.md

```diff
@@ -544,6 +537,39 @@ they will be passed directly as `read_csv` arguments. For example:
         encoding: latin-1
 ```
 
+#### map_remove_forbidden_characters
+
+The *map_remove_forbidden_characters* transformer allows the removal of forbidden characters from the values extracted from
+cells of the data frame. The pattern matching the characters that are *forbidden* characters should be passed to the 
+transformer as a regular expression. For example:
+
+```yaml
+    - map_remove_forbidden_characters:
+        columns:
+            - treatment
+        to_object: drug
+        via_relation: alteration_biomarker_for_drug
+        forbidden_characters: '[^0-9]' # Pattern matching all characters that are not numeric. 
+        # Therefore, you only allow numeric characters. 
+        substitute: "_" # Substitute all removed characters with an underscore, in case they are  
+        # located inbetween allowed_characters.
```

Comment:
Does the "inbetween" mean that forbidden characters located at the beginning or the end get treated differently?

Reply:
Yes, currently the characters that are forbidden but located at the beginning or the end of the name of the extracted cell value are stripped. But maybe that is not the best solution?

Reply:
I don't know. It seems a bit counter-intuitive, but I don't really have a strong opinion.